ui/core/components/results_viewer.tsx

```diff
@@ -37,19 +39,22 @@ export class ResultsViewer extends Component {
 		const pendingElemRef = ref<HTMLDivElement>();
 		const contentElemRef = ref<HTMLDivElement>();
 		const warningElemRef = ref<HTMLDivElement>();
+		const buttonElemRef = ref<HTMLDivElement>();
 
 		this.rootElem.appendChild(
 			<>
 				<div ref={pendingElemRef} className="results-pending">
 					<div className="loader"></div>
 				</div>
 				<div ref={contentElemRef} className="results-content"></div>
+				<div ref={buttonElemRef} className="button-zone" style="text-align: center"></div>
```

Comment:
There's a classname `text-center` (can you also change it for the warning-zone?

Reply:
Added the class for both.